src/components/category.js

```diff
@@ -220,9 +218,11 @@ Category.propTypes = {
   perLine: PropTypes.number.isRequired,
   emojiProps: PropTypes.object.isRequired,
   recent: PropTypes.arrayOf(PropTypes.string),
+  notFound: PropTypes.object.isRequired,
```

Comment:
You can be more specific here, by using `PropTypes.shape`: https://reactjs.org/docs/typechecking-with-proptypes.html